src/main/java/com/accelad/acctive/sim/kernel/core/evaluator/diff/BetweenStrictlyLessThanMaxFunction.java

```diff
@@ -30,7 +30,7 @@ public X getValue() {
         double a = firstBound.getValue().getReal();
         double b = secondBound.getValue().getReal();
 
-        if (v >= Math.min(a, b) && v <= Math.max(a, b)) {
+        if (v >= Math.min(a, b) && v < Math.max(a, b)) {
             return DFFactory.get(1d);
```

Comment:
I thought the reason would be something like that.
Unless we never ever going to check the return value against 1.0 or 0.0, then we probably need a translator of some sort (to hold the logic true => 1, false => 0 (or another :))